server/stream.go

```diff
@@ -249,6 +249,8 @@ type stream struct {
 	sources       map[string]*sourceInfo
 	sourceRetries map[string]*time.Timer
 
+	sourcesMirrorConsumerSetup *time.Timer
```

Comment:
Looks like java ;) Is this for sources as well as mirrors? 

`remoteConsumerSetup`

Reply:
Outaded after new rewrite